external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/FirstPollOffsetStrategy.java

```diff
@@ -31,12 +31,21 @@
      * topology deployment
      */
     LATEST,
+    /**
+     * The kafka spout polls records starting at the earliest offset whose timestamp is greater than or equal to the given startTimestamp.
```

Comment:
We should mention that these options only work for the Trident spout. I'd like to see this implemented for the regular spout as well. Are you up for adding support here, or would you rather leave it for someone else to handle later?

If we don't add support here, we need to mention in this javadoc that it doesn't work for the regular spout.

Reply:
I was hoping to get this in for the trident spout first. I had delayed it long enough :(. Adding the comment.

Reply:
That's fair, just wanted to be sure we noted somewhere that it doesn't work for the regular spout.